### DISCUSSION TITLE
W alphabet pattern in c language

### DISCUSSION THREAD

**Discussion by sjdjrbdj** [community/community#36400] [Code Security]
Hey bro or Siso I'm Nimit Shrikunj. My question is that plz tell me codes of alphabet of W pattern?Tq for read my question 😊

**Comment by DeepakKumarKhatri** [comment]
Here is the logic implemented in Python, you can convert it into C language

`

        for i in range(1):
            print("*" + 10 * " " + 4 * "*" + 10 * " " + "*")
        v = 1
        c = 4
        m = 2
        for j in range(5):
            print(v * " " + "*" + c * "  " + "*" + m * "  " + "*" + c * "  " + "*")
            v += 1
            c -= 1
            m += 1
`

**Comment by sjdjrbdj** [comment]
TQSM Sir for help me 😊
